package.json

```diff
@@ -86,10 +85,5 @@
       "^.+\\.jsx?$": "babel-jest",
       "^.+\\.pem$": "<rootDir>/tests/unit/__stubs__/fileTransformer.js"
     }
-  },
-  "config": {
```

Comment:
We're not really using `commitizen` since we've released version 2.0.0 of the Node API client. Also, it has a lot of vulnerable dependencies and doesn't seem to be actively maintained:
- `cz-conventional-changelog` => `5 Low | 5 Moderate | 1 High`

Reply:
I think @dabroek added `commitizen` in version 2 of the Node client. I'm okay with removing it if it's not being used.

Reply:
I am okay with removing it as well.